google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -193,18 +195,47 @@ private static String getAppEngineInstanceName() {
   }
 
   private static List<LoggingEnhancer> getEnhancers(Resource resourceType) {
-    List<LoggingEnhancer> enhancers;
+    List<LoggingEnhancer> enhancers = new ArrayList<>();
```

Comment:
Please consider switching to new ArrayList<>(2), as 2 is the maximum size of the list here.